portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About</title>
+    <link rel="stylesheet" href="about.css">
+    <script src="truths_and_lies.js"></script>
+  </head>
+  <body>
+    <div class="about_content">
+    <div id="header">
+        <a href="index.html"> <img src=/images/profile.jpg /> </a>
+        <div class="left">
+            <a href="#"> <p> About </p> </a>
+            <a href="survey.html"> <p> Survey </p> </a>
+            <a href="contact.html"> <p> Contact </p> </a>
+        </div>
+    </div>
+    <div id="about">
+        <h3>Want to get to know me? Let's play Two Truths One Lie</h3>
+        <div id="truth_and_lies_game">
+            <label id="question_num">Press Start to Begin!</label>
+            <div id="questions">
+
+                <!-- These fields will be populated by the about.js file -->
+
+                <div class="truth_and_lies_question">
+                    <label>
+                        <input type="radio" name="chosen_answer" id="first_answer" value="0">
+                        <img src="/images/two_truth_one_lie_imgs/stars.jpg" id="first_img">
```

Comment:
Hyphen instead of underscore here.

Also, with the comment directly below, maybe consider being a bit more specific in the name of what the image represents.  Something like first-claim-image would make it much more obvious upon reading the code that we are looking at the code representing the image tied to the first claim/statement.

Also, I hope you're ready for a thrilling in-depth exploration of Google variable naming with respect to abbreviations =D...
Notice that I'm using the full words "text" and "image" here and in my below comment... generally at Google, we tend to err on the side of somewhat longer and more clear variable names rather than abbreviations.  There's certainly some wiggle room where you can find your own preferences.  And we do have occasional abbreviations for words that are pretty obvious and can really cut down a long word (such as "id" and "comms" and "demo").  But things like text vs txt... is cutting out that one character worth making someone read three consonants in a row?  ...maybe?  To give that some context, I just did a search, and in the javascript code across all of our Payments department, there are around a thousand variables and functions with "text" in the name, and only one single instance of "txt".  And in searching CSS classes and IDs, I see 140 instances of "text" and zero instances of "txt".

Anyway, I ramble about this stuff because I love talking about tiny details, but it's certainly reasonable to disagree with me or Google style at large, especially if there's no specific rule written about it in the style guide.

Reply:
Changed to text and image + renamed to include claim